src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -101,6 +102,7 @@ void regmodule_offline_transformations(py::module m) {
         "compress_model_transformation",
         [](std::shared_ptr<ov::Model> model) {
             ov::pass::Manager manager;
+            manager.register_pass<ov::pass::DivisionByZeroFP16Resolver>();
```

Comment:
@mitruska, could you please explain - did we miss to trigger this transformation once we had implemented it?

Reply:
It was enabled in the legacy Python API, but not ported to the new Python API.
More details can be found in the comments for the ticket 66045.

Reply:
Not sure about only legacy API because this transformation is implemented in nGraph some time ago.
@mitruska, is this transformation not triggered now by MO? Please elaborate more because I remember the [PR](https://github.com/openvinotoolkit/openvino/pull/8676) with this transformation that I have reviewed. And we activated this transformation there. Was something incomplete? Thanks!

Reply:
The mentioned ([PR 8676](https://github.com/openvinotoolkit/openvino/pull/8676)) added the DivisionByZeroFP16Resolver to the legacy python API `compress_model_transformation`,
 then the legacy API has been removed ([PR 9121](https://github.com/openvinotoolkit/openvino/pull/9121/files)), and it was not added to the new python API for offline transformations in ([PR 8765](https://github.com/openvinotoolkit/openvino/pull/8765/files)).

Currently this transformation is not triggered at all.

Reply:
Okay, I got it. Thank you very much!